includes/options/class-amp-options-manager.php

```diff
@@ -603,7 +611,15 @@ public static function render_stories_deprecation_editor_notice() {
 						wp.data.dispatch( 'core/notices' ).createNotice(
 							'warning',
 							%s,
-							{ isDismissible: false }
+							{
+						  		isDismissible: false,
+						  		actions: [
+									{
+										url: 'https://amp-wp.org/documentation/amp-stories/exporting-stories/',
```

Comment:
(Apparently) links cannot be added inline, so I added the link to exporting Stories as an action.